src/reason_pprint_ast.ml

```diff
@@ -1992,19 +2140,15 @@ let formatCoerce expr optType coerced =
  *     );
  *
  *)
-let formatIndentedApplication delimiter headApplicationItem argApplicationItems =
+let formatIndentedApplication headApplicationItem argApplicationItems =
```

Comment:
I was able to remove the delimiter, though I see @yunxing had recently added a new location field inside of the delimiter. The test cases seem to do the right thing, so maybe not so important?